osd/KubePersistentVolumeFillingUp-customer-namespace.json

```diff
@@ -0,0 +1,9 @@
+
+{
+    "severity": "Warning",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Action required: update cluster monitoring configuration in ${NAMESPACE}",
+    "description": "Based on recent sampling, the PersistentVolume claimed by ${PVC} in Namespace ${NAMESPACE} is expected to fill up within ${DAYS} days. Currently ${PERCENT}% is available.SRE recommends reducing application logging on your cluster to ensure logging continues to function smoothly in the future.",
```

Comment:
> available.SRE
Add space after period

Reply:
@aweiteka It's for an app filling up the PV, coincidently the app was monitoring. I updated the PR, but now I am wondering if I should make the message more generic. 
My thinking was that a message about an app filling up a PV would be good to have in case this error becomes common as more users come.